### PR TITLE
display_line_section_impact_in_lines_api

### PR DESCRIPTION
**Pull request by xlqian**

### CODE REVIEW

File: documentation/rfc/line_sections.md [hunk 1]
```diff
@@ -140,7 +140,7 @@ This is for the calls api.navitia.io/v1/coverage/bob/<some pt ref filters>/traff
 
 This /traffic_reports api is meant to be a clever representation of the disruptions active on the object and thus we can do more things on this
 
-`/stop_areas/B/traffic_reports`  --> NO
+`/stop_areas/B/traffic_reports`  --> YES
```
Comment (antoine-de) [inline documentation/rfc/line_sections.md:143]: I think it should be NO

File: documentation/rfc/line_sections.md [hunk 2]
```diff
@@ -150,7 +150,7 @@ This /traffic_reports api is meant to be a clever representation of the disrupti
 
 `/stop_areas/A/routes/route_1_of_the_line_1/traffic_reports`  --> YES (I think NO would be better it but this will be quite difficult for not that much gain)
 
-`/stop_areas/C/routes/route_2_of_the_line_1/traffic_reports`  --> NO
+`/stop_areas/C/routes/route_2_of_the_line_1/traffic_reports`  --> YES
```
Comment (antoine-de) [inline documentation/rfc/line_sections.md:153]: I think it should be NO

File: documentation/rfc/line_sections.md [hunk 3]
```diff
@@ -109,9 +109,9 @@ We display only the stoppoints impacted by the line section
 `/stop_points/F_1_of_route_1_of_line_1`  --> NO
 
 
-`/routes/route_2_of_the_line_1`  --> NO
+`/routes/route_2_of_the_line_1`  --> YES
```
Comment (antoine-de) [inline documentation/rfc/line_sections.md:112]: maybe you can add 

```
`/routes/route_2_of_the_line_1`  --> YES (if the line_1 is displayed)
```

**Comment by xlqian** [comment]
this PR is a complicated solution, a new PR is opened. https://github.com/CanalTP/navitia/pull/2087

**Pull request closed without merging**